src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Appointee.java

```diff
@@ -1,32 +1,16 @@
 package uk.gov.hmcts.reform.sscs.ccd.domain;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Builder;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
```

Comment:
This doesn't really matter but if you are not using everything in the library you should only import what you need also does would you need the line below if u are importing all?